package.json

```diff
@@ -33,7 +33,7 @@
     "pkg-all": "source .circleci/local_publish_helpers.sh && generatePkgCli",
     "pkg-all-local": "yarn verdaccio-start && yarn verdaccio-connect && yarn publish-to-verdaccio && yarn pkg-all && yarn verdaccio-disconnect",
     "publish:master": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes",
-    "publish:dev": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --no-verify-access --yes",
+    "publish:dev": "lerna publish --preid=rc.$(git rev-parse --short HEAD) --pre-dist-tag rc --exact --include-merged-tags --no-verify-access --conventional-prerelease --conventional-commits --yes",
```

Comment:
I added `.` instead since semver tagging and npm will recognize it as a character to use in the version.

I would like for us to do some git tag magic to look at the previous version to increment up in the release candidate, but this works for now too.